bundler/helpers/lib/functions/force_updater.rb

```diff
@@ -1,5 +1,7 @@
 module Functions
   class ForceUpdater
+    class TransitiveDependencyError < StandardError; end
```

Comment:
Nice one, would be nice to also add a spec to cover this case so we don't accidentally break it in future.

Planning to rescue TransitiveDependencyError and raise a new update not possible error or similar?

Reply:
This code is already sort of tested, we currently catch _any_ error here: https://github.com/dependabot/dependabot-core/blob/5cb4b67e15e19afc0eb6fc377fca7b8098417477/bundler/lib/dependabot/bundler/update_checker/force_updater.rb#L62

Functionally nothing has changed yet, but I plan to follow up on this by actually handling this error differently, my plan is to add some context to the message that describes how the transitive dependency was locked. Totally happy to fold those changes into one PR, as that change should be tested separately 👍

Reply:
Ah gotcha, that makes sense 👍